### DISCUSSION TITLE
I can deploy with hexo g -d but it doesn't show up in my deployment or actions

### DISCUSSION THREAD

**Discussion by masterrite** [hexojs/hexo#5572] [Question / Answer] [answered]
I think I did something dumb. My page was working fine until I changed my repo to private. Since then, none of my deployments using hexo g -d is working. Here's what I get:
`$ hexo g -d
INFO  Validating config
INFO  Start processing
INFO  Files loaded in 451 ms
INFO  0 files generated in 199 ms
INFO  Deploying: git
INFO  Clearing .deploy_git folder...
INFO  Copying files from public folder...
INFO  Copying files from extend dirs...
On branch master
nothing to commit, working tree clean
Everything up-to-date
Branch 'master' set up to track remote branch 'master' from 'https://oauth2....`
But my actions and deployment pages do not show any update activities. Going to my website gives a 404 not found error. I didn't change anything in my local hexo folder or github repo except turning it into private. Turning it back to public doesn't help, either. Any help is appreciated. Thanks.

**Comment by uiolee** [comment, marked as answer]
https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site

Free GitHub accounts cannot use GitHub pages in private repositories.

You should learn how to enable github pages. This question has nothing to do with hexo
